<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerChoice</class>
 <widget class="QFrame" name="ServerChoice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
   <item>
    <widget class="QLabel" name="announcement">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="serverChoiceFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="serverChoice">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
         <item row="0" column="0" colspan="2">
          <widget class="QTableWidget" name="serverList">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string/>
            </property>
            <property name="toolTip">
             <string extracomment="&quot;This is to check if the server is password protected&quot;"/>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Server Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Players / Max</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Advanced connection</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Server &amp;description</string>
           </property>
           <property name="buddy">
            <cstring>description</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="PokeTextEdit" name="description">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="openExternalLinks">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLineEdit" name="nameEdit">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&amp;Trainer name</string>
           </property>
           <property name="buddy">
            <cstring>nameEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>&amp;Advanced connection</string>
           </property>
           <property name="buddy">
            <cstring>advServerEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="goBack">
           <property name="text">
            <string>&amp;Go back</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QPushButton" name="advancedConnection">
           <property name="text">
            <string>Advanced &amp;connection</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="advServerEdit"/>
           </item>
           <item>
            <widget class="QToolButton" name="savedServersList">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>10</horstretch>
               <verstretch>10</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>15</width>
               <height>15</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="iconSize">
              <size>
               <width>11</width>
               <height>11</height>
              </size>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonFollowStyle</enum>
             </property>
             <property name="autoRaise">
              <bool>false</bool>
             </property>
             <property name="arrowType">
              <enum>Qt::DownArrow</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PokeTextEdit</class>
   <extends>QTextBrowser</extends>
   <header>poketextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
